metals/src/main/scala/scala/meta/internal/metals/MetalsServerConfig.scala

```diff
@@ -31,6 +31,7 @@ final case class MetalsServerConfig(
     showMessage: ShowMessageConfig = ShowMessageConfig.default,
     showMessageRequest: ShowMessageRequestConfig =
       ShowMessageRequestConfig.default,
+    includeLastCharacter: Boolean = true,
```

Comment:
This behavior should not be configurable, I think it’s fine to enable it by default without an option to keep the old behavior.

Reply:
wouldn't this look weird for vim? The cursor at the end point to the next symbol, no?